app/models/programs.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Programs < ActiveRecord::Base
```

Comment:
Following Rails conventions, this class (and file name) should be `Program` (singular).

Reply:
10-4, let me fix that

Reply:
Updated it Mr. Holton

Reply:
No...?  Still says "Programs" and file is still named "programs.rb"